openapi-codegen/sbt-plugin/src/sbt-test/sbt-openapi-codegen/oneOf-json-roundtrip/Expected.scala.txt

```diff
@@ -160,18 +160,20 @@ object TapirGeneratedEndpoints {
 
 
 
-  type GetBinaryTestEndpoint = Endpoint[Unit, Unit, Unit, sttp.capabilities.pekko.PekkoStreams.BinaryStream, sttp.capabilities.pekko.PekkoStreams]
+  type GetBinaryTestEndpoint = Endpoint[String, Unit, Unit, sttp.capabilities.pekko.PekkoStreams.BinaryStream, sttp.capabilities.pekko.PekkoStreams]
   lazy val getBinaryTest: GetBinaryTestEndpoint =
     endpoint
       .get
       .in(("binary" / "test"))
+      .securityIn(auth.bearer[String]())
```

Comment:
Since both `BearerAuth` and `petstore_auth` (oauth2 implicit flow) translate to a bearer auth token at the api layer, this seemed reasonable when both were declared as the security for an endpoint. Not sure how to support different auth types, but would probably need to generate something like
```scala
      .securityIn(auth.bearer[Option[String]]())
      .securityIn(auth.apiKey(header[String]("api_key")))
      .mapSecurityIn(???)
```
with the appropriate `???` tbd. Not trying to go for thatt for now though.

Reply:
Yes, if there are multiple security inputs, optional values are the way to go: https://tapir.softwaremill.com/en/latest/docs/openapi.html#authentication-inputs-and-security-requirements
The `.map` there could instead be a `.mapDecode`, and if both are `None`, report a decoding error